github/WorkflowRun.py

```diff
@@ -318,6 +325,8 @@ def _useAttributes(self, attributes: dict[str, Any]) -> None:
             self._created_at = self._makeDatetimeAttribute(attributes["created_at"])
         if "updated_at" in attributes:  # pragma no branch
             self._updated_at = self._makeDatetimeAttribute(attributes["updated_at"])
+        if "actor" in attributes:  # pragma no branch
+            self._actor = self._makeClassAttribute(github.NamedUser.NamedUser, attributes["actor"])
```

Comment:
Need to import this class I guess

Reply:
Yeah wasn't too sure what is needed here. I saw later on `github.Repository.Repository` was referenced but only imported by name with `TYPE_CHECKING`.

Would adding `import github.NamedUser` be the right move?

Reply:
yes

Reply:
Addressed [841daed](https://github.com/PyGithub/PyGithub/pull/2764/commits/841daed3e18a5db15e9cdbc153ec505fb10d8147)